README.md

```diff
@@ -67,370 +49,40 @@ hash it and assign it to a Virtual Node. Riak fits really well here as it
 scales easily horizontally.
 This last use case is actually explained below.
 
-## More about Hashing and VNodes:
+## More about Hashing and VNodes
 Before performing an operation, a hashing function is applied to some data, a
 key. The key hash will be used to decide which node in the cluster should be
 responsible for executing the operation. The range of possible values the key
 hash can take (the keyspace, usually depicted as a ring), is partitioned in
 equally sized buckets, which are assigned to Virtual Vodes.
 
-![The Ring](ring.png)
+![The Ring](https://raw.githubusercontent.com/lambdaclass/riak_core_tutorial/master/ring.png)
 
 Virtual Nodes share what is called a keyspace.
 The number of VNodes is fixed at cluster creation and a given hash value will
 always belong to the same partition (i.e. the same VNode). The VNodes in turn
 are evenly distributed across all available physical nodes. Note this
 distribution isn't fixed as the keyspace partitioning is: the VNode distribution
 can change if a physical node is added to the cluster or goes down.
```

Comment:
you could end this conceptual section with a link to what to read next in the doc

Reply:
I've added a brief text that points to the tutorial, what do you think about it?